src/modules/workspaces/workspaces.usecase.spec.ts

```diff
@@ -949,4 +951,214 @@ describe('WorkspacesUsecases', () => {
+        manager: owner,
+        mainTeam: true,
+      });
+
+      const getWorkspace = jest
+        .spyOn(workspaceRepository, 'findOne')
+        .mockResolvedValue(Promise.resolve(workspace));
+
+      const workspaceUsers = jest
+        .spyOn(workspaceRepository, 'findWorkspaceUsers')
+        .mockResolvedValue(Promise.resolve([workspaceUser]));
+
+      const teamsAndMembers = jest
+        .spyOn(teamRepository, 'getTeamsAndMembersCountByWorkspace')
+        .mockResolvedValue(
+          Promise.resolve([
+            {
+              team: workspaceTeam,
+              membersCount: 1,
+            },
+          ]),
+        );
+
+      const fnGetMembers = service.getWorkspaceMembers(workspace.id, owner);
+      await expect(fnGetMembers).resolves.toMatchObject([{ isOwner: true }]);
+      await expect(fnGetMembers).resolves.toMatchObject([
+        { memberId: workspace.ownerId },
+      ]);
+
+      expect(getWorkspace).toHaveBeenCalled();
+      expect(workspaceUsers).toHaveBeenCalled();
+      expect(teamsAndMembers).toHaveBeenCalled();
+    });
+
+    it('When a user is not registered as Manager within a team then the "isManager" field will be False', async () => {
+      const user = newUser();
+      const workspaceUser = newWorkspaceUser({
+        workspaceId: workspace.id,
+        memberId: user.uuid,
+        member: user,
+      });
+
+      const workspaceUsers = jest
+        .spyOn(workspaceRepository, 'findWorkspaceUsers')
+        .mockResolvedValue(Promise.resolve([workspaceUser]));
+
+      const teamsAndMembers = jest
+        .spyOn(teamRepository, 'getTeamsAndMembersCountByWorkspace')
+        .mockResolvedValue([]);
+
+      const mockWorkspaceUsers: WorkspaceUserMemberDto[] = [
+        {
+          ...workspaceUser.toJSON(),
+          isOwner: false,
+          isManager: false,
+          freeSpace: workspaceUser.getFreeSpace(),
+          usedSpace: workspaceUser.getUsedSpace(),
+        },
+      ];
+
+      const fnGetMembers = service.getWorkspaceMembers(workspace.id, owner);
+
+      await expect(fnGetMembers).resolves.toStrictEqual(mockWorkspaceUsers);
+      await expect(fnGetMembers).resolves.toMatchObject([{ isManager: false }]);
+
+      expect(workspaceUsers).toHaveBeenCalled();
+      expect(teamsAndMembers).toHaveBeenCalled();
+    });
+
+    it('When a user is registered as Manager within a team then the "isManager" field will be True', async () => {
+      const user1 = newUser();
+      const user2 = newUser();
+
+      const workspaceUser1 = newWorkspaceUser({
+        workspaceId: workspace.id,
+        memberId: user1.uuid,
+        member: user1,
+      });
+      const workspaceUser2 = newWorkspaceUser({
+        workspaceId: workspace.id,
+        memberId: user2.uuid,
+        member: user2,
+      });
+
+      const workspaceTeam = newWorkspaceTeam({
+        workspaceId: workspace.id,
+        manager: user1,
+      });
+
+      newWorkspaceTeamUser({
```

Comment:
This might be something you missed to removed. You are not assigning it to any variable